### DISCUSSION TITLE
Radiator Valve (https://www.zigbee2mqtt.io/devices/TV02-Zigbee.html)

### DISCUSSION THREAD

**Discussion by effzeh972** [Koenkk/zigbee2mqtt#14296] [Devices]
Hi, I got myself this Radiator Valve (https://www.zigbee2mqtt.io/devices/TV02-Zigbee.html / purchase Link"https://www.amazon.de/dp/B0B8GP5D8K/ref=cm_sw_r_apa_i_Y8DF7VQV8PT7PSQENYD0_0?_encoding=UTF8&psc=1") because I thought it was a supported, but once I connected it with my network it shows up as unsopported. I already tried to get a converter for this Device to work, but since I´m very new to this thematik and never done that before I had no success creating one. My Question would be if someone has a Suggestion  or experience whit the Device what to do to get it to work.
Thanks ahead

**Comment by kcrussell** [comment]
I have the same problem. I purchased some TRVs which are labelled as Tuya TV02 and look identical to the illustration of the supported device on http://www.zigbee2mqtt.io/devices/TV02-Zigbee.html . I am using Zigbee2MQTT with Home Assistant, and when I pair the TV02 it appears on the Zigbee2MQTT device list as "unsupported", but with "Model" = TS0601.
Just to confuse things further, I notice that white label alternative IDs on the TV02 page http://www.zigbee2mqtt.io/devices/TV02-Zigbee.html refer to "TV01"!

**Reply by effzeh972** [reply]
Someone found the solution a few Days ago. (https://github.com/Koenkk/zigbee2mqtt/discussions/14320) with the right Converter, also it should now be in the Zigbee2MQTT Edge Add-on. But I can´t say it exactly because I added just the converter.

**Comment by kcrussell** [comment]
Thanks for your reply. I just got round to looking at this again, and found that my HA Zigbee2MQTT add-on was stuck on version 1.18 but there was no option to upgrade. After searching the forums, I discovered this was because the add-on version I originally installed is now frozen. I updated the repository (as explained here: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt#installation) and installed the latest version (1.28). This appears to have the updated integrations, and now recognises my TRV correctly.
